docs/data/date-pickers/adapters-locale/adapters-locale.md

```diff
@@ -151,6 +151,50 @@ This prop is available on all fields and pickers.
 You can control the field format spacing using the [formatDensity](/x/react-date-pickers/custom-field/#change-the-format-density) prop.
 :::
 
+#### Supported formats
+
+Some format might not yet be supported by the fields.
+For example, they don't support day of the year or quarter.
+
+Here is the list of the formats currently supported:
+
+- The year
+  - ✅ 2-digits values (e.g: `23`)
+  - ✅ 4-digits values (e.g: `2023`)
+  - ❌ Values with ordinal (e.g: `2023th`)
```

Comment:
For some reason, at least on MacOS the x is not presented in red color when rendered 🙈 🤷 
Do you have any idea what could be wrong? 🤔 
<img width="463" alt="Screenshot 2023-04-26 at 10 09 50" src="https://user-images.githubusercontent.com/4941090/234497555-33d0b455-8429-4e33-a409-4aa5f9a541a4.png">

Reply:
Same on Linux
If you have a better icon I'm Interested, I took the one of [this page](https://mui.com/x/react-data-grid/getting-started/#feature-comparison) but it's a little sad in grey

Reply:
IMHO, in that particular page, I don't mind the grey color as it reduces the noise, and when on a table, the idea is easier to grasp without a clear color difference. 🤔

Reply:
I've narrowed the reason behind this styling: The `fontFamily` is to blame. Removing the first `"IBM Plex Sans"` entry results in a red cross. 🙈